pkgs/development/tools/fastddsgen/default.nix

```diff
@@ -1,26 +1,35 @@
-{ lib, stdenv, runtimeShell, fetchFromGitHub, gradle_7, openjdk17 }:
-let
-  pname = "fastddsgen";
-  version = "3.3.0";
+{ lib
+, stdenv
+, makeWrapper
+, fetchFromGitHub
+, gradle_7
+, openjdk17
+}:
 
+let
   gradle = gradle_7;
-
 in
-stdenv.mkDerivation {
-  inherit pname version;
+
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
What is the actual benefit of `finalAttrs` here? You cannot override `version`, because this would require changing the `hash`. So it can be used only for changing `pname`. It might be useful to rename the package, but IMHO not much. Or am I missing something?

Reply:
There is no particular need for `finalAttrs` at this time, I just thought it looks cleaner. `finalAttrs` is now removed.